Cargo.toml

```diff
@@ -146,6 +146,10 @@ url = "2.2.2"
 rand = "0.8.4"
 rand_xoshiro = "0.6.0"
 hex = "0.4.3"
+tempdir = "0.3.7"
+# Needed to test SQLCipher
+libsqlite3-sys = { version = "*", default-features = false, features = ["bundled-sqlcipher"] }
```

Comment:
Try dropping `default-features = false`, it seems to be causing problems in CI.

Reply:
Done